src/app/issues-viewer/issues-viewer.component.ts

```diff
@@ -123,7 +123,10 @@ export class IssuesViewerComponent implements OnInit, AfterViewInit, OnDestroy {
       return of(false);
     }
 
-    return this.githubService.isRepositoryPresent(currentRepo.owner, currentRepo.name);
+    return (
+      this.githubService.isOrganisationPresent(currentRepo.owner) &&
+      this.githubService.isRepositoryPresent(currentRepo.owner, currentRepo.name)
```

Comment:
This function is no longer used, but for the sake of correctness, you may want to include the check on user as well.